src/miner_lora.erl

```diff
@@ -59,14 +60,18 @@ send(#packet_pb{payload=Payload, timestamp=When, signal_strength=Power, frequenc
 send_poc(Payload, When, Freq, DataRate, Power) ->
     gen_server:call(?MODULE, {send, Payload, When, Freq, DataRate, Power, false}, 11000).
 
+-spec port() -> {ok, inet:port_number()} | {error, any()}.
+port() ->
```

Comment:
Is this fun only here for debugging purposes? I don't see anything invoking it...

Reply:
look harder

Reply:
https://github.com/helium/miner/pull/343/files#diff-0a89925ece2208282952849cb72c97f2R663

Reply:
Ah yes :+1: